bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/items/ItemRegistry.java

```diff
@@ -68,16 +68,6 @@
     public Collection<Item> getItems(String pattern);
 
     /**
-     * Checks whether itemName matches the item name conventions.
-     * Item names must only consist out of alpha-numerical characters and
-     * underscores (_).
-     *
-     * @param itemName the item name to validate
-     * @return true, if the name is valid
-     */
-    public boolean isValidItemName(String itemName);
```

Comment:
Seems like openhab-core's org.openhab.core.items.internal.ItemUIRegistryDelegate is still [using this method](https://github.com/kaikreuzer/openhab-core/blob/e6107153e23f6393f2908b5dfcf368fc126454e7/bundles/org.openhab.core.compat1x/src/main/java/org/openhab/core/items/internal/ItemUIRegistryDelegate.java#L91). 
At least I have an error in eclipse after a clean setup...